### DISCUSSION TITLE
ScriptReference/EditorGUILayout.MaskField

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#21] [Notes]
# ScriptReference/EditorGUILayout.MaskField

https://docs.unity3d.com/ScriptReference/EditorGUILayout.MaskField.html

**Comment by vertxxyz** [comment]
If you want your `MaskField` to look like a button you might think to pass `GUIContent.none` to label and use a relevant `GUISkin`. The `MaskField` will still draw text of its own, the way to fix this is to make your own `GUIStyle` with [`imagePosition`](https://docs.unity3d.com/ScriptReference/GUIStyle-imagePosition.html) set to `ImagePostion.ImageOnly`.

**Comment by mvi** [comment]
Nothing and Everything are options that Unity adds to your supplied options with special integer values:  
0 = Nothing  
-1 = Everything - note using int.MaxValue will not display as Everything, instead it will show Mixed with all values set
